application/F3DConfigFileTools.cxx

```diff
@@ -1,5 +1,7 @@
 #include "F3DConfigFileTools.h"
 
+#include "F3DConfig.h"
```

Comment:
While we cant fail to build, I think we can improve our CI to actually test the binary config flag.

it would be as simple as adding a reference file to the "Check Install" step of our CI. This image should be using the installed config file.

Ok to do in another PT though.

Reply:
I have a better idea :)